### DISCUSSION TITLE
Notify push app not running on port 7867 causes container to be unhealthy

### DISCUSSION THREAD

**Discussion by YeapGuy** [nextcloud/all-in-one#1252] [Wiki]
### How to use GitHub
* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Is your feature request related to a problem? Please describe. 
I was about to open an issue like this:

> ### Steps to reproduce
> 1. Deploy Nextcloud AIO with the docker-compose method (I did this using Portainer, but I'm pretty sure it's not related to that)
> 2. Follow through the installation & configuration
> 3. Check status of `nextcloud-aio-nextcloud` container, either via some GUI, like Portainer, or `docker ps`
> 
> ### Expected behavior 
> `nextcloud-aio-nextcloud` container should be marked `HEALTHY`, because it works
> ### Actual behavior 
> `nextcloud-aio-nextcloud` container is marked `UNHEALTHY`
> 
> ### Host OS 
> DietPi v8.9.2
> 
> #### Nextcloud AIO version 
> 2.0.3
> 
> #### Current channel 
> latest-arm64
> 
> #### Other valuable info 
> Naturally, I tried to debug (and ideally fix) this myself, and here's what I discovered:
> 1. First, I researched how this Docker container health system works, and found out it's declared in the Dockerfile
> 2. I looked at the Dockerfile of the Nextcloud container in this repo, and found that the health check for it is [declared on the last line](https://github.com/nextcloud/all-in-one/blob/99cf7f385a21726e86f9a9713e3e7159cc3625b1/Containers/nextcloud/Dockerfile#L255), as such: `HEALTHCHECK CMD (nc -z localhost 9000 && nc -z localhost 7867) || exit 1`
> 3. I tried to see which part of the command is failing, and here's the results: ![image](https://user-images.githubusercontent.com/43749899/195403196-0a9b0bf7-faf4-48da-b876-d9820452c760.png)
> 4. I don't know what port 7867 is even used for, so I'm stuck here...

But then I dug into this repository more, found in the Caddy reverse proxy example config that port 7867 is used for some "Notify Push". I figured I'd check the container logs, maybe I find more information there? And I found some errors that resulted from invalid syntax in my config.php (which I forgot I even changed, my bad). So I fixed those, restarted, and boom! The container is suddenly healthy. THEN, I had an aha moment, and realized that notify-push is an app that only opens its port when Nextcloud has 0 errors.

It's a pretty ingenious way to notify users something is wrong, but it took me like 30 minutes to find out how it works.

### Describe the solution you'd like
Maybe this should be documented somewhere? To be honest, I'm only creating this issue to save time for others who might be frustrated by the container being unhealthy (they might not be technically capable enough to find out what's wrong by themselves too). Feel free to close this if there's no place you could document it.

**Comment by szaimen** [comment]
Lets move this to the wiki section then :)

**Reply by YeapGuy** [reply]
Ah, I didn't know this existed! Thank you, I'll probably edit my post later to be less like a story and more like a wiki page. 😄

**Comment by szaimen** [comment]
I think this should not happen anymore nowadays.
